### DISCUSSION TITLE
Add-On Installation Error

### DISCUSSION THREAD

**Discussion by NeverAGod** [windingwind/zotero-better-notes#664] [Q&A] [answered]
Hello,

I am getting an "Add-On Installation Error" when trying to enable this add-on. My software is up-to-date and I chose the most recent release of the Zotero Better Notes add-on. 

![image](https://github.com/windingwind/zotero-better-notes/assets/143440584/d8ead3c3-f250-4e0d-99e4-99be6a7b1da2)
![image](https://github.com/windingwind/zotero-better-notes/assets/143440584/0c934345-944d-4c3d-acdc-03568ca43076)

Thank you!
L

OS: Windows 10
Experience Level: New to Zotero

**Comment by windingwind** [comment, marked as answer]
See https://github.com/windingwind/zotero-better-notes#-install
